drivers/misc/ti-st/tty_hci.c

```diff
@@ -36,6 +36,7 @@
 #include <linux/interrupt.h>
 
 #include <linux/ti_wilink_st.h>
+#include "../../../net/bluetooth/led.h"
```

Comment:
can we do something like this? 

 #include <linux/ti_wilink_st.h>
-#include "../../../net/bluetooth/led.h"
+#include "led.h"
 
and 
--- a/drivers/misc/ti-st/Makefile
+++ b/drivers/misc/ti-st/Makefile
@@ -4,4 +4,8 @@
 #
 obj-$(CONFIG_TI_ST)            += st_drv.o
 st_drv-objs                    := st_core.o st_kim.o st_ll.o
+
+ccflags-y      := -Inet/bluetooth
 obj-$(CONFIG_ST_HCI)           += tty_hci.o